pages/lessons/fundamentals/wolovim-part-1.mdx

```diff
@@ -0,0 +1,321 @@
+      <Question question="eth-intro-1/3-checkin/Q5" />
+</SideDrawer> 
+<br/>
+
+Now you’re ready to surf the chain! That’s not a thing people say. I just made that up. Let’s take a quick tour.
+
+### **The quick tour**
+
+First things first, a sanity check:
+
+```python
+In [5]: w3.is_connected()
+Out[5]: True
+```
+
+Since we’re using the tester provider, this isn’t a very valuable test, but if it does fail, chances are you typed something in wrong when instantiating the `w3` variable. Double-check that you included the inner parentheses, i.e., `Web3.EthereumTesterProvider()`.
+
+### **Tour stop #1: accounts**
+
+As a convenience, the tester provider created some accounts and preloaded them with test ether. First, let’s see a list of those accounts:
+
+```python
+In [6]: w3.eth.accounts
+Out[6]: ['0x7E5F4552091A69125d5DfCb7b8C2659029395Bdf',
+ '0x2B5AD5c4795c026514f8317c7a215E218DcCD6cF',
+ '0x6813Eb9362372EEF6200f3b1dbC3f819671cBA69', ...]
+```
+
+If you run this command, you should see a list of ten strings that begin with `0x`. Each is a **public address** and is, in some ways, analogous to the account number on a checking account. You would provide this address to someone that wanted to send you ether.
+
+As mentioned, the tester provider has preloaded each of these accounts with some test ether. Let’s find out how much is in the first account:
+
+```python
+In [7]: w3.eth.get_balance(w3.eth.accounts[0])
+Out[7]: 1000000000000000000000000
+```
+
+That’s a lot of zeros! Before you go laughing all the way to the fake bank, recall that lesson about currency denominations from earlier. Ether values are represented in the smallest denomination, wei. Convert that to ether:
+
+```python
+In [8]: w3.from_wei(1000000000000000000000000, 'ether')
+Out[8]: Decimal('1000000')
+```
+
+One million test ether — still not too shabby.
+
+### **Tour stop #2: block data**
+
+Let’s take a peek at the state of this simulated blockchain:
+
+```python
+In [9]: w3.eth.get_block('latest')
+Out[9]: AttributeDict({
+   'number': 0,
+   'hash': HexBytes('0x9469878...'),
+   'parentHash': HexBytes('0x0000000...'),
+   ...
+   'transactions': []
+})
+```
+
+A lot of information gets returned about a block, but just a couple things to point out here:
+
+1. The block number is zero — no matter how long ago you configured the tester provider. Unlike the real Ethereum network, which includes a new block every 13 seconds, this simulation will wait until you give it some work to do.
+2. `transactions` is an empty list, for the same reason: we haven’t done anything yet. This first block is an **empty block**, just to kick off the chain.
+3. Notice that the `parentHash` is just a bunch of empty bytes. This signifies that it's the first block in the chain, also known as the **genesis** **block**.
+
+### **Tour stop #3: transactions**
+
+We’re stuck at block zero until there’s a transaction to mine, so let’s give it one. Send a few test ether from one account to another:
+
```

Comment:
I'm stuck here @wolovim 
I don't know how to get this code into the terminal;
` tx_hash = w3.eth.send_transaction({
   'from': w3.eth.accounts[0],
   'to': w3.eth.accounts[1],
   'value': w3.toWei(3, 'ether')
})`

Reply:
Hey Marc, this bit after line 267 is really the only thing holding me up. Are we supposed to be trying to enter the code into the terminal? In the lesson, I can see that there's an input, but no suggested output. 
 `tx_hash = w3.eth.send_transaction({ 
 'from': w3.eth.accounts[0], 
 'to': w3.eth.accounts[1], 
 'value': w3.toWei(3, 'ether') })`
 
 It possible to continue wiht the lesson without doing this step,but then the values after the transactions don't tally up to the lesson and my own terminal output. But that's just nitpicking. But the user will get stuck on the

Reply:
softened this language ("if you were to do this, the code would look like...") and corrected the additional `toWei` mention.